src/commands/branches.ts

```diff
@@ -100,6 +100,11 @@ export const builder = (argv: yargs.Argv) =>
             describe: 'Connect to a new branch via psql',
             default: false,
           },
+          annotation: {
+            type: 'string',
+            hidden: true,
```

Comment:
why hidden? we will not promote this option?

Reply:
I would not promote it at least until we have all the parts are updated and tested on prod.